app/scripts/controllers/detect-tokens.test.js

```diff
@@ -241,6 +240,8 @@ describe('DetectTokensController', function () {
         networkControllerMessenger,
         'NetworkController:stateChange',
       ),
+      onTokenListStateChange: (tokenListState) =>
+        console.log('onTokenListStateChange', tokenListState), // this is missing
```

Comment:
This is now a required field by the TokensController.
It's used like this in mobile: https://github.com/MetaMask/metamask-mobile/blob/2326d8707aeb01480d765bc417553b0517ab8411/app/core/Engine.ts#L299